ArchiSteamFarm/Steam/Interaction/Actions.cs

```diff
@@ -174,7 +174,7 @@ public ulong GetFirstSteamMasterID() {
 
 	/// <remarks>This action should be used if you require full inventory exclusively, otherwise consider calling <see cref="ArchiHandler.GetMyInventoryAsync" /> instead.</remarks>
 	[PublicAPI]
-	public async Task<(HashSet<Asset>? Result, string Message)> GetInventory(uint appID = Asset.SteamAppID, ulong contextID = Asset.SteamCommunityContextID, Func<Asset, bool>? filterFunction = null) {
+	public async Task<(HashSet<Asset>? Result, string Message)> GetInventory(uint appID = Asset.SteamAppID, ulong contextID = Asset.SteamCommunityContextID, string? language = null, Func<Asset, bool>? filterFunction = null) {
```

Comment:
This is a breaking change. We can make it non-breaking by moving the new argument to the end.